.ci/install_runtime.sh

```diff
@@ -65,7 +65,11 @@ if [ "$KATA_HYPERVISOR" = "nemu" ]; then
 fi
 
 if [ "$KATA_HYPERVISOR" = "qemu" ] && [ "$experimental_qemu" == "true" ]; then
+	runtime_repo="github.com/kata-containers/runtime"
+	runtime_repo_dir="$GOPATH/src/${runtime_repo}"
```

Comment:
useless variable? I guess you forgot to use it in the following lines?

Reply:
@devimc fix it